com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -122,8 +140,11 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             m_ForwardLights = new ForwardLights();
             //m_DeferredLights.LightCulling = data.lightCulling;
             this.m_RenderingMode = data.renderingMode;
+            this.m_DepthPrimingMode = data.depthPrimingMode;
             this.usesRenderPass = data.useNativeRenderPass;
 
+            this.m_CanUseDepthPriming = !SystemInfo.hasHiddenSurfaceRemovalOnGPU;
```

Comment:
We need to check if ARM has this flag and possibly disable for ARM. @manuele-bonanno ?

Reply:
yes storing MSAA sounds very expensive on ARM because of the lack of compression, according to them. 
Not sure about that flag for ARM GPUs. @hkr?

Reply:
hasHiddenSurfaceRemovalOnGPU is true for:
- Qualcomm Adreno 5 and up
- PowerVR
- Apple Silicon

whereas the internal hasTiledGPU is true for:
- Qualcomm Adreno
- PowerVR
- ARM Mali
- Vivante
- Apple Silicon

So it appears we're missing a few platforms by simply just checking for hasHiddenSurfaceRemovalOnGPU .

Reply:
Mali's HSR (FPK) is not as efficient as e.g. PowerVR.
Main use of `hasHiddenSurfaceRemovalOnGPU` so far was to decide if we should do front-to-back sort of opaque objects or not.
For Mali it's usually worth sorting, also because sorting is relatively cheap for many games.
For most games depth priming is probably significantly more expensive than sorting. So the trade-off will be different.